CHANGELOG.md

```diff
@@ -86,7 +111,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Integration with Digital Twin registry service.
 
 
-[Unreleased]: https://github.com/catenax-ng/tx-dft-backend/compare/dft-backend-1.8.0...main
+[Unreleased]: https://github.com/catenax-ng/tx-dft-backend/compare/dft-backend-1.9.0...main
+[1.9.0]: https://github.com/catenax-ng/tx-dft-backend/compare/dft-backend-1.9.0...dft-backend-1.8.1
```

Comment:
These links do point to a fork and not your upstream repository.
Changelog is created for your upstream. Did you ensure, that the all the mentioned released do actually exist?

Reply:
Sure I am checking.

Reply:
Asked team to review from committer.
Change log updated as per the upstream.